Testing/SystemTests/tests/framework/LiquidsReflectometryReductionTest.py

```diff
@@ -111,3 +112,119 @@ def runTest(self):
+    """
+        Test algorithm LiquidsReflectometryReduction with simple binning
+    """
+    def runTest(self):
+        # '/SNS/users/wzz/Mantid_Project/builds/testing/sf_185602_Si2InDisk_auto.cfg'
+        scaling_factor_file = FileFinder.getFullPath("sf_185602_Si2InDisk_auto.cfg")
+
+        error_weighting = False
+        r = LiquidsReflectometryReduction(RunNumbers=[185746],
+                                          NormalizationRunNumber=185610,
+                                          SignalPeakPixelRange=[133, 145],
+                                          SubtractSignalBackground=True,
+                                          SignalBackgroundPixelRange=[129, 149],
+                                          NormFlag=True,
+                                          NormPeakPixelRange=[135, 143],
+                                          NormBackgroundPixelRange=[132, 146],
+                                          SubtractNormBackground=True,
+                                          ErrorWeighting=error_weighting,
+                                          LowResDataAxisPixelRangeFlag=True,
+                                          LowResDataAxisPixelRange=[75, 165],
+                                          LowResNormAxisPixelRangeFlag=True,
+                                          LowResNormAxisPixelRange=[115, 142],
+                                          TOFRange=[9965.86996219, 23253.6965784],
+                                          TOFSteps=40,
+                                          IncidentMediumSelected='Si2InDisk',
+                                          GeometryCorrectionFlag=False,
+                                          QMin=0.005,
+                                          QStep=0.02,
+                                          AngleOffset=-0.008,
+                                          AngleOffsetError=0.001,
+                                          ScalingFactorFile=scaling_factor_file,
+                                          SlitsWidthFlag=True,
+                                          ApplyPrimaryFraction=True,
+                                          SlitTolerance=0.02,
+                                          PrimaryFractionRange=[5, 295],
+                                          OutputWorkspace='_reflectivity_error')
+
+        self._success = self.verify_result(r)
+
+    def validate(self):
+        return self._success
+
+    def verify_result(self, test_output):
+        """Verify reduced result
+
+        Parameters
+        ----------
+        test_output: Workspace2D
+
+        Returns
+        -------
+        bool
+            same or not
+
+        """
+        test_vec_x = test_output.readX(0)
+        test_vec_y = test_output.readY(0)
+        test_vec_e = test_output.readE(0)
+
+        gold_data = np.array([
+            [4.459747512458072521e-02, 4.190316438906013260e-05, 6.788916891170880312e-06],
```

Comment:
I bothered to look at this datablock as given in [Steps to Reproduce](https://code.ornl.gov/sns-hfir-scse/reflectometry/reflectometry/-/issues/78#steps-to-reproduce) and I verified in that section of the Defect issue that the datablock with `ErrorWeighting=True` has identical  numbers to the datablock with `ErrorWeighting=False`. In other words, we don't know how from these two datablocks how many significant figures are necessary to set in `numpy.testing.assert_allclose`.   
Please run this test with `ErrorWeighting=True` and verify for yourself how different the numbers come out

Reply:
`run this test with ErrorWeighting=True`: The test is failed as being expected.